### DISCUSSION TITLE
Custom Module execution

### DISCUSSION THREAD

**Discussion by andromedatsa** [synthetichealth/synthea#1120] [Q&A] [answered]
Dear Team,

I would like to create a custom module using the module builder and execute the patients creation based on it. However, while i am able to create the module, i am still missing some configurations & lookups that i need to have to generate the patients. Is there any help available on how to achieve this? Any information will be very helpful! Thank you in advance!

**Comment by jawalonoski** [comment, marked as answer]
Table-based lookup transitions are documented here: https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-Transitions#table

You can see examples of it in use in the AML, COVID, and HIV modules.

For example, from the `acute_myeloid_leukemia.json` module:

```json
    "Determine_Levofloxacin_Prophylaxis": {
      "type": "Simple",
      "lookup_table_transition": [
        {
          "transition": "Levofloxacin",
          "default_probability": 0.2638,
          "lookup_table_name": "AML.csv"
        },
        {
          "transition": "No_Levofloxacin_Prophylaxis",
          "default_probability": 0.7362,
          "lookup_table_name": "AML.csv"
        }
      ]
    },
```
The AML csv file https://github.com/synthetichealth/synthea/blob/master/src/main/resources/modules/lookup_tables/AML.csv

**Reply by andromedatsa** [reply]
Thank you, will check this out.
